src/main/java/mServer/crawler/sender/MediathekNdr.java

```diff
@@ -322,6 +337,16 @@ private void filmSuchen_1(String strUrlThema, String thema, String titel, String
                 // src="/fernsehen/hallondsopplatt162-player_image-2c09ece0-0508-49bf-b4d6-afff2be2115c_theme-ndrde.html"
                 // http://www.ndr.de/fernsehen/hallondsopplatt162-ppjson_image-2c09ece0-0508-49bf-b4d6-afff2be2115c.json
                 // id="pp_hallondsopplatt162"
+                if (datum.isEmpty()) {
+                    String tmp = seite2.extract("<span itemprop=\"datePublished\"", "</");
+                    if ((pos1 = tmp.indexOf(">")) != -1) {
+                        tmp = tmp.substring(pos1 + 1, tmp.length());
+                        String[] dateValues = parseDateTime(tmp, strUrlThema);
+                        datum = dateValues[0];
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "datum". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1226)